roles/reproducer/tasks/configure_architecture.yml

```diff
@@ -4,13 +4,13 @@
   block:
     - name: Push script
       ansible.builtin.copy:
-        dest: "/home/zuul/deploy-va.sh"
+        dest: "/home/zuul/deploy-architecture.sh"
         mode: "0755"
         owner: "zuul"
         group: "zuul"
         content: |-
           pushd src/github.com/openstack-k8s-operators/ci-framework
```

Comment:
there's an issue with the script, apparently calling it via `command` isn't working, I suspect the missing shebang to be the issue - currently testing.

Reply:
corrected.